NSS-Day-Cohort-42/rare-media-moguls
# fixed app views

#### Changes Made
1. reverted app views to old setup

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-fix-app-views
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py server
```
4. Test app functionality.
> when user is logged in everything should work as intended.
> check app views and make sure it's the right component
5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
